extensions/wikia/AffiliateService/AffiliateService.i18n.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$messages = array();
+
+$messages['en'] = array(
+	'affiliate-service-disclaimer' => 'Fandom may earn an affiliate commission on sales made from links on this page.',
+);
+
+$messages['qqq'] = array(
```

Comment:
Is this right? qqq?

Reply:
I did have a question about that. I'm wondering if it's a default fallback.

Reply:
`qqq` is a dummy language used for message descriptions, it does not actually display in the interface.

Reply:
It's a way of documenting messages.

https://www.mediawiki.org/wiki/Localisation#Message_documentation